internal/tokenizer/tokenize.go

```diff
@@ -8,12 +8,12 @@ import (
 	"gopkg.in/src-d/enry.v1/regex"
 )
 
-// Tokenize returns language-agnostic lexical tokens from content. The tokens
-// returned should match what the Linguist library returns. At most the first
-// 100KB of content are tokenized.
+// Tokenize returns lexical tokens from content. The tokens returned should match what
```

Comment:
[optional] I suggest making the comment correct-as-intended and separating corrections into a removable paragraph, e.g.,

```
// Tokenize returns lexical tokens from content. The tokens returned match what
// the Linguist library returns. At most the first ByteLimit bytes of content are tokenized.
//
// BUG: Until https://github.com/src-d/enry/issues/193 is resolved, there are some
// differences between this function and the Linguist output.
```

Rationale: This makes it clear what the intent is, and makes it trivial to update the comment when the fix is in: Just delete the BUG paragraph.

(This is not crucial, however—what you wrote is also fine).

Reply:
Thank you Michael, that does make a lot of sense and thank you for a concrete suggestions!